internal/api/http.go

```diff
@@ -232,12 +237,20 @@ func newHeaderRoundTripper(authToken string, headers map[string]string, rt http.
 		a += ", application/vnd.github.shadow-cat-preview"
 		headers[accept] = a
 	}
-	return headerRoundTripper{headers: headers, rt: rt}
+	return headerRoundTripper{host: host, headers: headers, rt: rt}
 }
 
 func (hrt headerRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	// If request is not for the host we have configured, do not add authorization headers.
+	if !isValidHost(req.URL.Host, hrt.host) && hrt.headers[authorization] != "" {
```

Comment:
Please use `u.Hostname()` instead of `u.Host`, since the former strips out the port name if present

Reply:
Thanks for this. I have modified as suggested.